### DISCUSSION TITLE
What's the focus?

### DISCUSSION THREAD

**Discussion by KalleHallden** [EXERLOG/exer_log#138] [General]
I thought I could create this so that we can discuss what should be focused on. I'll try to update this discussion with whatever I am working on currently. That way if something more important comes up then let me know here. 

(may not be the best idea but thought it might work)

**Comment by KalleHallden** [comment]
I will be working on creating the [product flow map](https://docs.google.com/document/d/172ofqs4MEg5aDWP-tBBRm6QFhPE9R7KzAaDSs3C4wAA/edit?usp=sharing) so that we can have some idea of where we're going and what needs to be done.

Edit: any feedback on the document would be helpful. I updated the link above to link to the document I have written so far.

**Reply by momshaddinury** [reply]
That's great. Leave the PR review to me and @jorre127. We will make sure to leave comments on what happened and why we did something. 

Issues are going to pile up. Nothing to do there. Most rarely takes time to go through existing issues before creating a new one. So we need to let it be for now.

Aside from that, I will focus on the Workout page and see if I can organize the code a bit better and then try to migrate it to MVVM.

**Reply by KalleHallden** [reply]
Perfect! That sounds like a plan

**Comment by momshaddinury** [comment]
@jorre127 made two tiny adjustments (without PR):
1. https://github.com/EXERLOG/exer_log/commit/0e0c1f51ff8163ca8e6fceda22190e94aaf36cf2
2. https://github.com/EXERLOG/exer_log/commit/3494c0508b82159cc7f45d2c874dc3eb810bdbb1

**Comment by momshaddinury** [comment]
@KalleHallden little update from my side.

1. The routing issue has been fixed. 
2. The responsive issue with the calendar widget is solved.
3. The whole calendar feature code moved to **src/feature/calendar** directory
4. The code is simplified
5. Some unused files (authentication.dart and email_signup.dart) is removed
